client/src/js/apis/fetchData.js

```diff
@@ -0,0 +1,55 @@
+// client/src/js/apis/fetchData.js
+
+import fetch from 'cross-fetch';
+
+const roomUrl = 'http://localhost:3003/rooms/';
+
+const getRoomInfo = async (roomId, NumberReviewsPerPage) => {
+  try {
+    const url = new URL(roomUrl + roomId);
+    const params = { pageonly: 0, start: 0, limit: NumberReviewsPerPage };
```

Comment:
should pageonly be camel case style?

Reply:
I'm not sure if the restful api will be case sensitive or not, so I just put them all in lower case for now.